appserver/web/web-core/src/main/java/org/apache/catalina/realm/JAASRealm.java

```diff
@@ -55,19 +55,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-// Portions Copyright [2019] Payara Foundation and/or affiliates
+// Portions Copyright [2019-2024] Payara Foundation and/or affiliates
```

Comment:
License specifically under GPLv2 for GlassFish port.
```suggestion
// Portions Copyright [2019-2024] Payara Foundation and/or affiliates
// Payara Foundation and/or its affiliates elects to include this software in this distribution under the GPL Version 2 license
```